icicle-compiler/src/Icicle/Sea/FromAvalanche/Program.hs

```diff
@@ -188,9 +226,9 @@ seaOfStatement state compute stmt
      KeepFactInHistory _
       -> Pretty.empty
   where
-   stNew   n = "s->" <> stateInputNew (seaOfName n)
-   stRes   n = "s->" <> stateInputRes (nameOfResumable compute $ seaOfName n)
-   stHas   n = "s->" <> stateInputHas (nameOfResumable compute $ seaOfName n)
+   stNew   n = "s->" <> stateInputNew (prettyText . takeSeaName $ n)
```

Comment:
I wanted to make it harder to accidentally mangle a c name multiple names (resulting in name mismatch between definition and uses). `takeSeaName . mangleToSeaName` hides this imo